### ISSUE TITLE
T90 | items spelled incorrectly? 

### ISSUE THREAD

**Issue by IftekharUddin** [RenegadeLucien/project-tenacity#67]
some items i try to type in like "Wizard robe top" and "Bronze med helm" return an error that the item doesn't exist. I understand that sometimes the capitalization is off but some items just result in this error. I'm not sure if their names are abbreviated or have some different name under the covers but It makes it difficult to use.

**Comment by RenegadeLucien** [comment]
Yes, many pieces of armor were left out of the armor database. These were usually either untradeables or, in this case, armors that had strict equals or betters (Zamorak robe (top) and Bronze full helm, respectively.)

I added both pieces into the database. Will go out in next release.

**Comment by IftekharUddin** [comment]
Ah i see. That makes sense. I was just curious about that section because that gets factored into the calculation of what to do next right? If so, if I only have the lesser versions of the armour and don't know any better, how do I use this more optimally? Or is it just entering in more and more items? If so, where exactly in the code do you add items? Maybe I could help with a pull request in the future.

**Comment by RenegadeLucien** [comment]
It's pretty much just a question of adding more items.

**Comment by ZarrZarr** [comment]
Running into issues with a lot of low level gear pieces, that don't have many if at all any counter parts (Imp Horn Set, Black Armor Set, Mithril Weapons)
